frontend/src/components/core/Button.tsx

```diff
@@ -41,6 +41,12 @@ export const PrimaryButton = styled(DefaultButton)<any>`
   }
 `;
 
+export const SecondaryRedButton = styled(PrimaryButton)<any>`
```

Comment:
Is this any tag being used? If not I think we can get rid of it

```suggestion
export const SecondaryRedButton = styled(PrimaryButton)`
```

Reply:
I don't believe so, yeah we can get rid of that.